src/hardware/configuration.rs

```diff
@@ -105,6 +105,32 @@ pub struct PounderDevices {
 /// Static storage for the ethernet DMA descriptor ring.
 static mut DES_RING: ethernet::DesRing = ethernet::DesRing::new();
 
+/// Setup ITCM and load its code from flash
+unsafe fn setup_itcm() {
+    extern "C" {
+        static mut __sitcm: u32;
+        static mut __eitcm: u32;
+        static mut __siitcm: u32;
+    }
+    use core::{ptr, slice, sync::atomic};
+
+    // ITCM is enabled on reset on our CPU but might not be on others.
+    // Keep for completeness.
+    const ITCMCR: *mut u32 = 0xE000_EF90usize as _;
```

Comment:
Ideally, these would be added to `cortex-m-rt::peripheral::SCB`. It looks like they don't exist for now though, so this is fine as-is. Maybe make it `0xE00_EF90_usize`?

Reply:
That would be wrong.

Reply:
I'm not a fan of the additional underscore before the type. Looks like it's commonly done without. E.g. https://doc.rust-lang.org/rust-by-example/primitives/literals.html
There is no underscore after the 0x either.

Reply:
While I acknowledge that my original comment had a typo (which was fixed), my comment was around the underscore betweeo EF90 and usize. I still haven't decided which form I favor more, but in all honesty, no one is going to be passively glancing at this, and it's easy enough to read when you actually do care.